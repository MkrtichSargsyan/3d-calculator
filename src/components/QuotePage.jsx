import React from 'react';

function QuotePage() {
  return (
    <div>
      It has always been said that wealth cannot buy you joy, but it can buy you
      marijuana and that is just enough.
    </div>
  );
}

export default QuotePage;
