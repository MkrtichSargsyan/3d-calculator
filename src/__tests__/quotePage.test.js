import renderer from 'react-test-renderer';
import QuotePage from '../components/QuotePage';

it('expect to render quote page', () => {
  const result = renderer.create(<QuotePage />).toJSON;
  expect(result).toMatchSnapshot();
});
