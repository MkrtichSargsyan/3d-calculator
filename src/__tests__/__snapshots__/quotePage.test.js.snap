// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render quote page 1`] = `[Function]`;
